### ISSUE TITLE
How to debug PyLint in GitLab CI?

### ISSUE THREAD

**Issue by Bengt** [PyCQA/pylint#7967]
### Question

When running PyLint on my code base in GitLab CI, I get this ominous error:

```bash
$ venv/bin/python3.10 -m tox -e pylint
pylint: commands[0]> pylint --rcfile=.pylintrc.ini --recursive=y img_neural/ tests/ setup.py
pylint: exit 2 (0.00 seconds) /builds/iml/vr/image/img.neural> pylint --rcfile=.pylintrc.ini --recursive=y img_neural/ tests/ setup.py
  pylint: FAIL code 2 (0.06=setup[0.06]+cmd[0.00] seconds)
  evaluation failed :( (0.13 seconds)
```

### Documentation for future user

I would have expected PyLint to print some explicit errors for me to fix.

### Additional context

PyLint runs fine on my code base locally:

```
$ venv/bin/python3.10 -m tox -e pylint
[...]
  pylint: commands succeeded
  congratulations :)
```

My tox environment definition:

```ini
[testenv:pylint]
allowlist_externals = bash
skip_install = true
deps =
    -rrequirements.d/pylint.in
commands =
    pylint \
        --rcfile=.pylintrc.ini \
        --recursive=y \
        img_neural/ tests/ setup.py
    bash -c \
        "python -m pip freeze --all > requirements.d/pylint.txt"
```

**Comment by Bengt** [comment]
This works for me:

```ini
[testenv:pylint]
allowlist_externals = bash
skip_install = true
deps =
    -rrequirements.d/pylint.in
commands =
    -   pylint \
        --rcfile=.pylintrc.ini \
        --recursive=y \
        img_neural/ tests/ setup.py
    -   bash -c \
        "python -m pip freeze --all > requirements.d/pylint.txt"
```

It seems like the commands need not only to be divided by a newline, but also by a `-`.
